### ISSUE TITLE
Create guest views for a personal blog instance

### ISSUE THREAD

**Issue by ivanminutillo** [bonfire-networks/bonfire-app#846]
- [ ] Homepage / profile page
- [ ] Posts / Threads page

**Comment by ivanminutillo** [comment]
added a basic /write page with a full-page composer, linked from the user more menu in the left sidebar

**Comment by kamazeuci** [comment]
I think this would be wonderful but I wouldn't focus it on only "personal" blogs, but maybe something more like "minimal website feature for persons, organizations or other institutions.

Nowadays, a lot of companies and institutions don't even have a website, because Instagram (lousily) gives them a similar function. AFAIK The fediverse is still lacking that function and that could help us bring more people into it.

 I would do something like this:

https://github.com/bonfire-networks/bonfire-app/issues/1849

**Comment by ivanminutillo** [comment]
I totally agree, focusing on slashpages would be a big win both in terms of maintainability (building a cms has its own challenges, UX and so forth) - it would require time and dedication, not a feature we can have halfbacked. Slashpages can be built in progressively, they're much more to maintain and already fits very well with the fediverse UX

**Comment by DeadSuperHero** [comment]
I'm very much in favor of this. Something we're looking at for a future project would be to create a Bonfire site that features a Ghost-like experience for reading and publishing. Dedicated views for articles would be a huge deal!
